### PR TITLE
fix：Handling test issues and others

### PR DESCRIPTION
**Pull request by wzh425**
# Description
fix：Handling test issues
chore：Update scheduler
refactor：ASPNETCORE_URLS http=>https
refactor：No sending policy is set. Do not leave the scheduler

### CODE REVIEW

File: src/Services/Masa.Mc.Service/Masa.Mc.Service.Admin.csproj [hunk 1]
```diff
@@ -39,7 +39,7 @@
 		<PackageReference Include="Masa.BuildingBlocks.Authentication.Identity" Version="$(MasaContribPackageVersion)" />
 		<PackageReference Include="Masa.Contrib.Isolation.MultiEnvironment" Version="$(MasaContribPackageVersion)" />
 		<PackageReference Include="Masa.Contrib.Isolation.UoW.EFCore" Version="$(MasaContribPackageVersion)" />
-		<PackageReference Include="Masa.Contrib.ReadWriteSpliting.Cqrs" Version="$(MasaContribPackageVersion)" />
+		<PackageReference Include="Masa.Contrib.ReadWriteSpliting.Cqrs" Version="0.6.0-preview.3" />
```
Comment (doddgu) [inline src/Services/Masa.Mc.Service/Masa.Mc.Service.Admin.csproj:42]: ?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Mc.Service/Masa.Mc.Service.Admin.csproj (hunk 1 above)
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Masa.Mc.Service.Admin.csproj:42]: 最高版本就是0.6.0-preview.3

Comment (doddgu) [reply src/Services/Masa.Mc.Service/Masa.Mc.Service.Admin.csproj:42]: 包名换了 Masa.Contrib.ReadWriteSplitting.Cqr

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**